cpu/fe310/context_frame.c

```diff
@@ -0,0 +1,83 @@
+
```

Comment:
This file and some others have an empty line at the top, which kinda jumps into the eye. Please fix.

Reply:
Fixed that in context_frame.h and context_frame.c